dell/precision/5560/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  config,
+  lib,
+  ...
+}: {
+  imports = [
+    ../../../common/pc/laptop
+    ../../../common/pc/laptop/ssd
+    ../../../common/cpu/intel
+    ../../../common/gpu/intel/tiger-lake
+    ../../../common/gpu/nvidia/prime.nix
+    ../../../common/gpu/nvidia/turing
+  ];
+
+  hardware.enableRedistributableFirmware = lib.mkDefault true;
+
+  boot = {
+    blacklistedKernelModules = ["nouveau"];
+    kernelModules = ["kvm-intel"];
+    kernelParams = ["i915.modeset=1"];
+  };
+
+  hardware = {
+    graphics = {
+      enable = lib.mkDefault true;
```

Comment:
Is this not enabled by the nvidia module?

Reply:
you're right, I've deleted it now